<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgServiceScheduler</class>
 <widget class="QDialog" name="DlgServiceScheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="yearComboBox"/>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Aplicar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Año</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="monthComboBox"/>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="printButton">
     <property name="text">
      <string>Imprimir</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Exportar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="unselectAllButton">
     <property name="text">
      <string>Desmarcar todos</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="selectAllButton">
     <property name="text">
      <string>Marcar todos</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Mes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="6" colspan="4">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="actionComboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tipo de acción</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>yearComboBox</tabstop>
  <tabstop>monthComboBox</tabstop>
  <tabstop>selectAllButton</tabstop>
  <tabstop>unselectAllButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>printButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
